Mathlib/Algebra/Order/Monoid/OrderDual.lean

```diff
@@ -88,10 +88,8 @@ instance orderedCommMonoid [OrderedCommMonoid α] : OrderedCommMonoid αᵒᵈ :
 @[to_additive OrderDual.OrderedCancelAddCommMonoid.to_contravariantClass]
 instance OrderedCancelCommMonoid.to_contravariantClass [OrderedCancelCommMonoid α] :
     ContravariantClass αᵒᵈ αᵒᵈ HMul.hMul LE.le where
-    -- Porting note: We need to specify the implicit arguments here because of
-    -- https://github.com/leanprover/lean4/issues/1892
-    -- We should be able to remove this after nightly-2022-11-30 arrives.
-    elim a b c := @OrderedCancelCommMonoid.le_of_mul_le_mul_left α _ a c b
+    -- Porting note: need to specify `α`; not sure why
```

Comment:
This seems unsurprising since we're proving a statement about `αᵒᵈ`. I'd remove the note entirely